cmd/lastZWriter/lastZWriter.go

```diff
@@ -87,6 +113,51 @@ func fastaFinder(lastZ string, pairwise string, reference string, species string
 	return theseLines
 }
 
+// fastaFinder_Simple has no matrix string
+// fastaFinder_Simple has a different output file name structure: ref.species/qName/tName.qName.axt
+func fastaFinder_Simple(lastZ string, pairwise string, reference string, species string, par string) (lines []string) {
+	var currLine string
+	var theseLines []string
+	var tMatches, qMatches, tFiles, qFiles []string
+	tPath := pairwise + "/" + reference + ".byChrom"
+	qPath := pairwise + "/" + species + ".byChrom"
+
+	if _, e := os.Stat(tPath); os.IsNotExist(e) {
+		log.Fatalf("There is no .byChrom directory for the target (reference) species.")
+	}
+	if _, e := os.Stat(qPath); os.IsNotExist(e) {
+		log.Fatalf("There is no .byChrom directory for the query species.")
+	}
+
+	tMatches, _ = filepath.Glob(tPath + "/*.fa")
+	qMatches, _ = filepath.Glob(qPath + "/*.fa")
+
+	for tF := range tMatches {
+		_, tName := filepath.Split(tMatches[tF])
+		tFiles = append(tFiles, tName)
+	}
+	for qF := range qMatches {
+		_, qName := filepath.Split(qMatches[qF])
+		qFiles = append(qFiles, qName)
+	}
+	for t := range tFiles {
+		tName := strings.TrimSuffix(tFiles[t], ".fa")
```

Comment:
Depending on where you go, I've seen fasta filesi n the suffices `.fa/fasta/.mfa` point being people do different things. One suggestion is to add a check to trip the suffix. If you decide to go this route, you could then expand on the glob command on line 132-133

```suggestion
		if strings.HasSuffix(tFiles[t], ".fa") {
			tName := strings.TrimSuffix(tFiles[t], ".fa")
		} else strings.HasSuffix(tFiles[t], ".fasta") {
			tName := strings.TrimSuffix(tFiles[t], ".fasta")
		} else {
			log.Errorf("Error: there is a bug in the glob command, we found files of other suffixes!!")
		}
```